setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+# 201 and 202 are spaces after ( and before ), 501 is line length. These
+# are exceptions allowed (encouraged?) by Galaxy style guidelines.
```

Comment:
Huh, ok. Could there be documentation on all of the conventions? (Maybe in Contributing.md?)